ddl/ddl_db_test.go

```diff
@@ -334,6 +332,7 @@ func (s *testDBSuite) TestCancelAddIndex(c *C) {
 			checkErr = errors.Trace(err)
 			return
 		}
+		jobIDs := []int64{job.ID}
```

Comment:
Why this is better?

Reply:
It's accurate job‘s ID. 
Original test is failed but doesn't assert. So we can't find the job's ID is wrong.